rotkehlchen/db/dbhandler.py

```diff
@@ -1145,21 +1145,27 @@ def add_blockchain_accounts(
             account_data: list[BlockchainAccountData],
     ) -> None:
         # Insert the blockchain account addresses and labels to the DB
-        blockchain_accounts_query = []
-        address_book_query = []
+        blockchain_accounts_query, bindings_to_update, bindings_to_delete = [], [], []
         for entry in account_data:
             blockchain_accounts_query.append((entry.chain.value, entry.address))
-            if entry.label is not None:
-                address_book_query.append((entry.address, entry.chain.value, entry.label))
+            if entry.label not in [None, '']:
```

Comment:
```suggestion
            if entry.label:
```

this way, you don't need the else branch.